### PR TITLE
Enable more rubocops

### PR DESCRIPTION
**Pull request by bjk-soundcloud**
* Style/LeadingCommentSpace
* Style/SpaceAfterControlKeyword
* Style/SpaceBeforeBlockBraces
* Style/SpaceInsideBlockBraces
* Style/SpaceInsideHashLiteralBraces
  - SupportedStyles: space
* Style/EmptyLines
* Style/EmptyLinesAroundAccessModifier
* Style/EmptyLinesAroundBody
* Style/LineEndConcatenation

Disable Style/ParenthesesAroundCondition

### CODE REVIEW

**Comment by bjk-soundcloud** [comment]
I removed the travis changes from this branch.

File: bin/lhm-kill-queue [hunk 1]
```diff
@@ -115,10 +114,10 @@ module Lhm
     def kill_process(process_id)
       puts "killing #{ select_statement(process_id) }"
 
-      if(@confirm)
+      if @confirm
         print "confirm ('y' to confirm): "
 
-        if(gets.strip != 'y')
+        if (gets.strip != 'y')
```
Comment (arthurnn) [inline bin/lhm-kill-queue]: why it didnt remove the parenthesis in here?

**Comment by arthurnn** [comment]
besides my comment, everything looks good.. Are you done with this? or still working on it?
About travis changes, we can do in another PR.

File: bin/lhm-kill-queue (hunk 1 above)
Comment (bjk-soundcloud) [inline bin/lhm-kill-queue]: You mean on #117?  This is probably due to the way rubocop auto-fixes work.  It has this nasty tendency to fix unrelated style issues, even if they're disabled. :-(  I will put it back and re-run the rubocop test.

**Comment by bjk-soundcloud** [comment]
Yea, this one is good enough for now.  I will send another PR or so with additional rubocop fixes when I have more time. :-)

**Comment by arthurnn** [comment]
great.. thanks

**Pull request merged**